lighthouse-core/audits/dobetterweb/no-datenow.js

```diff
@@ -57,7 +57,7 @@ class NoDateNowAudit extends Audit {
     const pageHost = url.parse(artifacts.URL.finalUrl).host;
     // Filter usage from other hosts.
     const results = artifacts.DateNowUse.usage.filter(err => {
-      return url.parse(err.url).host === pageHost;
+      return url.parse(err.url || '').host === pageHost;
```

Comment:
Agreed. Line/col number without an URL makes them less useful, so let's keep what you have then. It looks like it finds all the violations on www.inflearn.com just fine :)

![screen shot 2016-11-02 at 10 51 54 am](https://cloud.githubusercontent.com/assets/238208/19940596/72eb9a2a-a0ea-11e6-95e8-5d4eef9de08c.png)